specs/dispute-game-interface.md

```diff
@@ -146,41 +189,52 @@ interface IDisputeGame {
 
     /// @notice Getter for the game type.
     /// @dev `clones-with-immutable-args` argument #1
     /// @dev The reference impl should be entirely different depending on the type (fault, validity)
     ///      i.e. The game type should indicate the security model.
-    /// @return _gameType The type of proof system being used.
-    function gameType() external view returns (GameType _gameType);
+    /// @return gameType_ The type of proof system being used.
+    function gameType() external view returns (GameType gameType_);
 
     /// @notice Getter for the root claim.
-    /// @return _rootClaim The root claim of the DisputeGame.
+    /// @return rootClaim_ The root claim of the DisputeGame.
     /// @dev `clones-with-immutable-args` argument #2
-    function rootClaim() external view returns (Claim _rootClaim);
+    function rootClaim() external view returns (Claim rootClaim_);
 
     /// @notice Getter for the extra data.
     /// @dev `clones-with-immutable-args` argument #3
-    /// @return _extraData Any extra data supplied to the dispute game contract by the creator.
-    function extraData() external view returns (bytes memory _extraData);
+    /// @return extraData_ Any extra data supplied to the dispute game contract by the creator.
+    function extraData() external view returns (bytes memory extraData_);
 
-    /// @notice Returns the address of the `BondManager` used 
-    function bondManager() public view returns (IBondManager _bondManager);
+    /// @notice Returns the address of the `BondManager` used
+    function bondManager() public view returns (IBondManager bondManager_);
 
     /// @notice If all necessary information has been gathered, this function should mark the game
     ///         status as either `CHALLENGER_WINS` or `DEFENDER_WINS` and return the status of
     ///         the resolved game. It is at this stage that the bonds should be awarded to the
     ///         necessary parties.
     /// @dev May only be called if the `status` is `IN_PROGRESS`.
-    function resolve() public returns (GameStatus _status);
+    /// @return status_ The status of the game after resolution.
+    function resolve() public returns (GameStatus status_);
+
+    /// @notice A compliant implementation of this interface should return the components of the
+    ///         game UUID's preimage provided in the cwia payload. The preimage of the UUID is
+    ///         constructed as `keccak256(gameType . rootClaim . extraData)` where `.` denotes
+    ///         concatenation.
+    /// @return gameType_ The type of proof system being used.
+    /// @return rootClaim_ The root claim of the DisputeGame.
+    /// @return extraData_ Any extra data supplied to the dispute game contract by the creator.
+    function gameData() external view returns (GameType gameType_, Claim rootClaim_, bytes memory extraData_);
 }
 
 ////////////////////////////////////////////////////////////////
```

Comment:
QQ: Why is the output attestation dispute game part of this interface by the fault dispute game isn't?

Reply:
The Fault Dispute Game is part of the interface, we just don't document it here in the spec. The interface is [quite larger](https://github.com/ethereum-optimism/optimism/blob/55dc82b802865996ef83db1fa5802bd9e45e542c/packages/contracts-bedrock/src/dispute/interfaces/IFaultDisputeGame.sol) and I feel adding it in here detracts users from understanding the FDG.

Reply:
Given we're not doing the attestation dispute game anymore should we remove it from the specs?

Reply:
Yeah, it should be removed. I'll do so in this pr